purchases-sample/src/main/res/layout/package_card.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        xmlns:bind="http://schemas.android.com/apk/res-auto"
+        xmlns:tools="http://schemas.android.com/tools">
+
+    <data>
+
+        <variable
+                name="currentPackage"
+                type="com.revenuecat.purchases.Package" />
+    </data>
+
+    <com.google.android.material.card.MaterialCardView
+            android:id="@+id/package_card"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_margin="@dimen/padding_small"
+            android:padding="@dimen/padding_small"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent">
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+                android:id="@+id/package_card_content"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:padding="@dimen/padding_small">
+
+            <androidx.appcompat.widget.AppCompatTextView
+                    android:id="@+id/package_product_title"
+                    style="@style/TextAppearance.AppCompat.Medium"
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:text="@{currentPackage.product.title}"
+                    app:layout_constraintEnd_toStartOf="@+id/package_card_barrier"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toTopOf="parent"
+                    tools:Text="Introductory Price (PurchasesSample)" />
+
+            <androidx.appcompat.widget.AppCompatTextView
+                    android:id="@+id/package_product_description"
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_marginStart="@dimen/padding_tiny"
+                    android:text="@{currentPackage.product.description}"
+                    app:layout_constraintEnd_toStartOf="@+id/package_card_barrier"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toBottomOf="@+id/package_product_title"
+                    tools:text="Product with an introductory price" />
+
+            <androidx.appcompat.widget.AppCompatTextView
+                    android:id="@+id/package_identifier"
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_marginStart="@dimen/padding_tiny"
+                    android:text="@{currentPackage.identifier}"
```

Comment:
thinking about this, maybe the package identifier won't be the most useful thing - it'll usually be something like `$rc_monthly`, which isn't particularly helpful when debugging. 

I think it might be more useful to display the product identifier instead.

And maybe the package type, and if the package type is custom, do something like `Custom: "my_custom_type"`. 

What do you think?

Reply:
yeah sure :) update coming..